rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/StatisticsView.java

```diff
@@ -79,7 +78,7 @@ protected String getCellCSSText(ListGridRecord record, int rowNum, int colNum) {
                 String base = super.getCellCSSText(record, rowNum, colNum);
                 if (colNum == 2) {
                     String num = record.getAttribute("count");
-                    if (Integer.parseInt(num) == 0) {
+                    if (num.contentEquals("0")) {
```

Comment:
The Integer.parseInt(num) threw an exception if it couldn't parse the num string. 

Since the line checks if there is a 0 in the string, I decided to check it with contentEquals, that way it is possible to have a string like "None" inside the num string without having an exception thrown.